.travis.yml

```diff
@@ -22,3 +22,15 @@ jobs:
     - python: "3.6"
     - python: "3.7"
     - python: "3.8"
+    - stage: Deploy
+      script: skip
+      python: 3.8
+      deploy:
+        provider: pypi
+        user: aws-cloudformation-developers
+        password:
+          secure: ACHz2AGFnR0QWjwRzO3mpO0lnM/WTEQVtY0p8cbf3CrbwJW/DzqAD/IwFWC4dK0CND2VObYuQb0n8mLQKw5k9Bx464NThOu4CEZhRj7oUvMLXAgTAMOsi8GDMMRv7iyBeEvOs5D2cSeLtZEL2sPd3hXcsEG6Mb03DWG66CM3K1/MHmsnKI1dlqgvzzJb2i6dmM2ZAxW90/fAW9Qc4qLLLg93K3vvWm4y+XdOBukf554vPZxRAcEJ4ywjf/e3J1oGDDqs1uWgO4klrZJOjnxaeM+cL1IRCldq1ITQgyczoqBH8eM6QaT0cccs8GiM4z+Le15rwvahm2JV418XbsGrmL39gabLwZarKaVXwlEmLq0LVs0uXoi4RzYfQTULh0vlUprm/4+zhKLEg2YC1fy0dz2V7FB4n2RrC7T72sjbjXtWgTCfw/V+G6DJ2omvTSUsD9oG6L9L0FZ3f0sQV4uvQJ0tzyij+EsbZJ6FFPe71VPCXvTUsFHLN3xRkTSSR2OJg7gO6MmUhvWjewUo+QX9LY6EszU5POZ4Yg4jfeftpJDXUN/dX2pUSr6NuWNvYjf9J5OYvfTtOV6eD4a+2GWLRs7F1pKXVo5K4VCFhIe3qwZh4IF8S5nFUyUVq7WG/CLAh0d6oRIuetKMTzNNrTh1j4vpEVbVofOTDi7BmkIICCo=
+        on:
+          tags: true
+        distributions: sdist bdist_wheel
+        skip_existing: true
```

Comment:
https://docs.travis-ci.com/user/deployment/pypi/
https://docs.travis-ci.com/user/encryption-keys/
https://docs.travis-ci.com/user/build-stages/

https://github.com/aws-cloudformation/cfn-python-lint/blob/eecd57e946ae3ee69067d9f6861467dd9b0a0698/.travis.yml#L6-L8

https://github.com/aws-cloudformation/cfn-python-lint/pull/508

@kddejong is the `stages` section required to make sure the tests pass before deploying? Although I suppose it's safe to assume the tests already passed if it's already been merged:

```
By default, Travis CI will only release from the master branch.
Builds triggered from Pull Requests will never trigger a release.
```

Reply:
Yea.  We wanted to validate the tests worked (a final time) before release.  It was also a final check to validate any outside dependencies didn't cause us any issues.  You're right though by the time we do the actual release we have tested the code about 3 times (once on the PR, once when its merged to main, and then again at release).  Obviously this a little redundant.  Your needs may also be dependent on if you are doing branch protection.  We are protecting against direct modifications to the main branch and making sure PRs are tested with an in sync main branch. If you aren't doing those things than you may want to keep those tests.

Reply:
I agree with @kddejong I would rather have redundant checks vs miss something. We do have branch protection from direct modification. I would still prefer to keep this change. It is not time sensitive functionality and should not cause any issues by running the tests again.